scripts/unit-tests.sh

```diff
@@ -31,6 +31,7 @@ set -e
 
 PAVER_ARGS="-v"
 PARALLEL="--processes=-1"
+export SKIP_NPM_INSTALL="True"
```

Comment:
This environment variable should be added to `tox.ini`'s `passenv` to remain present for pytest calls inside tox virtualenvs.

Reply:
🤒 good catch

Reply:
I guess this technically isn't needed since we run this entire script inside tox, but still seems like it would eventually come back to bite us if we didn't list it there.